views/view_listings.erb

```diff
@@ -11,6 +28,11 @@
       <a href="/space?id=<%= space.id %>"><%= space.name %></a>
       <%= space.description %>
       <%= space.price %>
+      <a target="_blank" href="<%= space.url %>">
```

Comment:
i wonder if this should click through to the space page (as the title does above) rather than redirecting to the image url?